packages/dumb/src/beforeEach.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2022-2023, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import yourloopsEn from '../../../locales/en/yourloops.json'
+import translationEn from '../../../locales/en/translation.json'
+import parameterEn from '../../../locales/en/parameter.json'
+import i18n from 'i18next'
+import { initReactI18next } from 'react-i18next'
+import locales from '../../../locales/languages.json'
+
+export const i18nOptions = {
+  lng: 'en',
+  keySeparator: false,
+  nsSeparator: '|',
+  returnEmptyString: false,
+  react: {
+    useSuspense: true,
+    transSupportBasicHtmlNodes: true // allow <br/> and simple html elements in translations
+  },
+  ns: locales.namespaces,
+  defaultNS: locales.defaultNS,
+  fallbackNS: locales.fallbackNS,
+  resources: {
+    en: { yourloops: yourloopsEn, main: translationEn, params: parameterEn }
+  }
+}
+
+i18n.use(initReactI18next)
+i18n.init(i18nOptions).then(() => {
```

Comment:
Is the `.then()` mandatory, given that it does nothing?

Reply:
It's not mandatory but the IDE gives me a warning if I don't have it